sda/webservice.go

```diff
@@ -0,0 +1,81 @@
+package sda
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"reflect"
+
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/network"
+	"github.com/dedis/protobuf"
+	"golang.org/x/net/websocket"
```

Comment:
I believe this file is an old one and is not supposed to be there anymore right ? Still using golang websocket implementation.